jte-jsp-converter/src/test/java/gg/jte/convert/jsp/JspToJteConverterTest.java

```diff
@@ -247,6 +254,7 @@ void givenSetup(Consumer<Converter> setup) {
     void whenJspTagIsConverted(String jspTag, String jteTag) {
         JspToJteConverter converter = new MyConverter();
         converter.convertTag(jspTag, jteTag, c -> {
+            c.setPutParametersOnSeparateLines(true);
```

Comment:
This should be the default setting for the converter. Otherwise we're now having tests for a configuration the default user does not experience, which is bad.

Reply:
Should I remove the `setPutParametersOnSeparateLines` method in the first place? Or keep it so that the end user can turn it off (it's on by default)?

Reply:
I would remove the option. The converter may come with a sensible default setting how stuff is formatted (now, tag parameters on new lines), but it is not a formatter, so we should not put too much formatting logic in here. If different formatting is required, this effort is better put in the IntelliJ plugin.

Reply:
Rather there should probably be a command line tool for formatting instead of relying on the plugin, so it can also be applied in CI?

Reply:
> Rather there should probably be a command line tool for formatting

Good point, but it's outside of the scope of this issue. Option removed.